tmt/steps/execute/simple.py

```diff
@@ -0,0 +1,134 @@
+import os
+import tmt
+import shutil
+import click
+
+# Simple runner script name
+RUNNER = 'run.sh'
+
+
+class ExecuteSimple(tmt.steps.execute.ExecutePlugin):
+    """ Execute tests using the simple executor """
```

Comment:
The selected plugins' help should be displayed, right? Or is that not the case @psss?

Do we have only one `--help` for all plugins?

Reply:
Ah indeed:

```
tmt run execute -h shell.simple --help
Usage: tmt run execute [OPTIONS]

  Execute shell tests using the simple executor

  A simple 'run.sh' script is run directly on the guest, executes all
  tests in one batch and checks the exit code for the test results.
```

Anyway, this might not be obvious, that the `--help` is affected with some other option, in our case `--how`. @psss could we add examples to the `execute` help page for pointer to this? i.e.

```
$ tmt run execute --help
Usage: tmt run execute [OPTIONS]

  Run tests using the specified framework

  Supported methods:

  shell.simple ......... Execute shell tests using the simple executor
  beakerlib.simple ..... Execute beakerlib tests using the simple executor
 
  To see additional help of a plugin, provide it via `--how` option, e.g. `tmt run execute -h shell.simple --help`
  

```

Reply:
Good idea, will add an explicit mention to the main help.

Reply:
Updated. Here's an example usage message:

```
> tmt run execute --help
Usage: tmt run execute [OPTIONS]

  Run tests using the specified framework

  Supported methods:

  shell.simple ......... Execute shell tests using the simple executor
  beakerlib.simple ..... Execute beakerlib tests using the simple executor

  Use 'tmt run execute --help --how <method>' to learn more about given
  execute method and all its supported options.

Options:
  -h, --how METHOD  Use specified method for test execution.
  -v, --verbose     Show more details. Use multiple times to raise verbosity.
  -d, --debug       Provide debugging information. Repeat to see more details.
  -q, --quiet       Be quiet. Exit code is just enough for me.
  -f, --force       Force overwriting existing stuff.
  -n, --dry         Run in dry mode. No changes, please.
  --help            Show this message and exit.
```